src/staff_augmentation/components/Header/index.js

```diff
@@ -1,16 +1,55 @@
+          <path d="M183.096 126.64C180.825 128.937 177.945 130.538 174.793 131.259C171.641 131.979 168.345 131.789 165.289 130.711C162.234 129.632 159.544 127.71 157.532 125.166C155.521 122.623 154.271 119.564 153.927 116.344C153.763 114.769 153.816 113.18 154.085 111.622C154.204 110.924 154.366 110.234 154.572 109.557C155.376 106.918 156.809 104.516 158.749 102.558C158.846 102.46 158.941 102.365 159.035 102.269C161.662 99.7379 165.035 98.1177 168.658 97.646C172.282 97.1744 175.966 97.8763 179.17 99.6487C180.387 100.332 181.519 101.157 182.543 102.106L182.907 102.467C184.507 104.047 185.781 105.927 186.657 107.997C187.532 110.068 187.992 112.289 188.01 114.534C188.027 116.779 187.602 119.003 186.759 121.081C185.916 123.158 184.672 125.047 183.096 126.64Z" fill="#15579E"/>
+          <path d="M221.457 97.8954C192.793 87.9188 186.414 63.2339 186.414 63.2339C202.842 54.2459 220.349 47.3745 238.518 42.7831C238.518 42.7831 237.085 66.0704 221.457 97.8954Z" fill="#AFC0E0"/>
+          <path d="M119.203 204.243C114.76 206.914 110.152 209.543 105.38 212.13C103.107 213.367 100.791 214.59 98.4326 215.799C98.4326 215.799 81.086 216.214 69.194 186.879C69.194 186.879 70.386 184.332 72.7866 179.891C74.6077 176.545 77.1203 172.13 80.3407 166.919C80.3407 166.919 84.8014 177.806 95.7275 188.613C102.454 195.312 110.432 200.624 119.203 204.243Z" fill="#AFC0E0"/>
+          <path d="M105.38 212.129C103.107 213.366 100.791 214.589 98.4324 215.798C98.4324 215.798 81.0858 216.213 69.1939 186.878C69.1939 186.878 70.3858 184.331 72.7864 179.891C72.784 179.888 87.8859 206.774 105.38 212.129Z" fill="#E3E8F4"/>
+          <path opacity="0.2" d="M179.195 99.6387L179.15 99.6318L159.057 102.257C158.963 102.353 158.869 102.448 158.772 102.546C156.832 104.504 155.398 106.906 154.594 109.545L182.572 102.101L179.195 99.6387Z" fill="white"/>
+          <path opacity="0.2" d="M171.865 112.697L153.938 116.335C153.774 114.76 153.827 113.171 154.096 111.612L171.847 110.922L171.865 112.697Z" fill="white"/>
+          <path opacity="0.2" d="M73.8204 191.458C73.8204 191.458 77.4945 183.611 84.9742 171.505C101.421 144.873 136.292 97.6199 191.053 67.8202C205.847 59.7347 221.517 53.3559 237.762 48.8061C238.381 44.9358 238.51 42.7777 238.51 42.7777C220.344 47.3712 202.84 54.2442 186.415 63.2333C131.655 93.033 96.784 140.286 80.3368 166.918C72.8571 179.025 69.1829 186.871 69.1829 186.871C73.8844 198.459 79.4338 205.403 84.4273 209.567C80.7873 205.376 77.1017 199.544 73.8204 191.458Z" fill="white"/>
+          <defs>
+            <linearGradient id="paint0_linear" x1="69.1874" y1="244.672" x2="39.6809" y2="212.279" gradientUnits="userSpaceOnUse">
+              <stop stop-color="#4776E6"/>
+              <stop offset="1" stop-color="#8E54E9"/>
+            </linearGradient>
+          </defs>
+        </svg>
+        <a href="#contact" className={classes.scrollDown}>
+          <SVGIcon name='nav_bar/chevron_right' className={classes.chevron} />
+        </a>
       </div>
     </section>
```

Comment:
- Please move the Contact link to outside of the content `div`
- Use `HashLink` from `react-router-hash-link` (already installed) to make the scrolling animated.

```suggestion
      </div>
      <HashLink to="#contact" smooth className={classes.scrollDown}>
        <SVGIcon name='nav_bar/chevron_right' className={classes.chevron} />
      </HashLink>
    </section>
```